### PR TITLE
Fix dask 2021.3.1

### PR DESCRIPTION
**Pull request by ericpre**
It would be good to fix the existing packages. Dask 2021.3.1 dropped support for python 3.6.

### CODE REVIEW

**Comment by ericpre** [comment]
@cjmartian, @chenghlee, not sure it this is right place to do this, but the dask-core package is broken on the defaults channels.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: dask-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
-  sha256: 566054b493d63c15732f2a640382b21e861571d61639f59341bc7695a9be138e
+  sha256: 1fa7e1809e7c50c5505297b03e6ee161e0695df73b4c4051b5b1c8fd670bf5d7
 
 build:
   number: 0
```
Comment (chenghlee) [inline recipe/meta.yaml:13]: Could you please bump the build number so we can rebuild without squashing the existing package? For the existing build 0, we'll apply a repodata hotfix.

**Review by chenghlee — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (ericpre) [reply recipe/meta.yaml:13]: Yes, makes sense, I needed to make another PR because this branch is not my fork!

**Comment by ericpre** [comment]
Closing for #2 with the build number bumped.

**Pull request closed without merging**